project-keeper/src/main/java/com/exasol/projectkeeper/validators/files/CiBuildWorkflowGenerator.java

```diff
@@ -0,0 +1,51 @@
+package com.exasol.projectkeeper.validators.files;
+
+import static com.exasol.projectkeeper.validators.files.FileTemplate.Validation.REQUIRE_EXACT;
+import static java.util.stream.Collectors.joining;
+
+import com.exasol.projectkeeper.shared.config.BuildConfig;
+
+class CiBuildWorkflowGenerator {
+    private final BuildConfig buildConfig;
+
+    CiBuildWorkflowGenerator(final BuildConfig buildConfig) {
+        this.buildConfig = buildConfig;
+    }
+
+    FileTemplate createReleaseDroidPrepareOriginalChecksumWorkflow() {
+        return new FileTemplateFromResource(".github/workflows/release_droid_prepare_original_checksum.yml",
```

Comment:
We could consider if we want to introduce a builder here as currently the `REQUIRE_EXACT` is followed by some replace operations which reads a little bit strangely in my eyes:

```java 
FileTemplate.builder()
    .fromResource("path/to.yml") // 
    .replacing("a", "b") // 
    .validation(REQUIRE_EXACT) //
    .build();
```

Reply:
Good idea! I propose to implement this in a separate PR.